src/test/java/com/microsoft/azure/functions/worker/binding/tests/RpcHttpRequestDataSourceTest.java

```diff
@@ -41,14 +56,150 @@ public static RpcHttp getTestRpcHttp(Object inputBody) throws Exception {
     return httpBuilder.build();
   }
 
+  @Test
+  public void rpcHttpDataSource_To_HttpRequestMessage_NullableQueryParamsEmpty_EnvSettingEnabled() throws Exception {
+    DefaultClassLoaderProvider classLoader = new DefaultClassLoaderProvider();
+    JavaFunctionBroker broker = new JavaFunctionBroker(classLoader);
+    FunctionEnvironmentReloadRequestHandler envHandler = new FunctionEnvironmentReloadRequestHandler(broker);
+    Map<String, String> existingVariables = System.getenv();
+    Map<String, String> newEnvVariables = new HashMap<>();
+    newEnvVariables.putAll(existingVariables);
+    newEnvVariables.put("FUNCTIONS_WORKER_NULLABLE_VALUES_ENABLED", "true");
+    envHandler.setEnv(newEnvVariables);
+    Method httpRequestMessageStringBodyMethod = getFunctionMethod("HttpRequestStringBody");
+    Map<String, String> queryMap = new HashMap<String, String>() {{
+      put("name", "");
+    }};
+    Parameter[] parameters = httpRequestMessageStringBodyMethod.getParameters();
+    String sourceKey = "testRpcHttp";
+    RpcHttp input = getTestRpcHttp(null, null, queryMap);
+    RpcHttpRequestDataSource rpcHttp = new RpcHttpRequestDataSource(sourceKey, input);
+    Optional<BindingData> actualBindingData = rpcHttp.computeByName(sourceKey,
+            parameters[0].getParameterizedType());
+    BindingData actualArg = actualBindingData.orElseThrow(WrongMethodTypeException::new);
+    HttpRequestMessage<?> requestMsg = (HttpRequestMessage<?>) actualArg.getValue();
+    assertEquals(requestMsg.getQueryParameters().get("name"), "");
```

Comment:
let's follow the convention that expect value comes first and test value comes second. I think we can use `assertEmpty` here.